src/edu/csus/ecs/pc2/ui/EditProblemPane.java

```diff
@@ -5340,8 +5434,10 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
      */
     private CustomValidatorSettings getCustomValidatorSettingsFromFields() {
 
+        CUSTOM_VALIDATOR_TYPE cvt = getCustomValidatorTypeFromUI();
+        
         // make sure exactly one of the Interface mode buttons is selected
-        if (!(this.getUsePC2ValStdRadioButton().isSelected() ^ this.getUseClicsValStdRadioButton().isSelected())) {
+        if (cvt == CUSTOM_VALIDATOR_TYPE.CV_NONE) {
```

Comment:
This change does not (I don't think) produce the correct effect.  The effect of the original code (the XOR in the `if` statement) was to show a message (and throw an exception) if it was the case that (somehow) either NONE of the "which interface to use" buttons were selected, or instead MORE THAN ONE of the buttons was selected.   This PR adds a third button -- but it removes the test which verifies that exactly one (of the now three) buttons is selected, replacing it with a test to see whether the "custom validator type" (which is really the "custom validator _interface_ type, as noted in another comment elsewhere) is NONE.  That may be an important test -- but it has removed the other test in the process.

I suppose one could argue that since the buttons are put in a `ButtonGroup`, they are automatically mutually exclusive (meaning, selecting one automatically deselects the others) -- which is true.  But that doesn't guarantee that one gets selected in the first place.  Currently, the code _does_ cause one to get selected.  But if some future programmer messes around with the code and (accidently or unknowingly) deletes the code which initially selects a button, then removal of this test will cause that condition to go undetected.

Reply:
No action being taken.  Code works as-is.  If we find a problem later for some unknown reason, we can create an issue.